### DISCUSSION TITLE
Sayı dizisi sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#242] [Zeka Sorusu] [answered]
9 – 4 – 8 – ? – 4 – 9 – 5

Soru işaretli yere ne gelmelidir?

**Comment by AhmetEmirErsoz** [comment, marked as answer]
pazartesi 9
salı 4
çarşamba 8
perşembe 8 yani cevap 8
cuma 4
cumartesi 9
pazar 5
